sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableAsyncClient.java

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.data.tables;
+
+import com.azure.core.annotation.ServiceClient;
+import com.azure.core.http.rest.PagedFlux;
+import reactor.core.publisher.Mono;
+
+import java.util.Map;
+
+@ServiceClient(
+    builder = TableClientBuilder.class,
+    isAsync = true)
+public class TableAsyncClient {
+    String tableName;
+
+
+    TableAsyncClient(String tableName) {
+        this.tableName = tableName;
+    }
+
+
+    /**
+     * returns the table associated with this table client
+     *
+     * @param tableName the name of the table
+     * @return the table
+     */
+    public Mono<AzureTable> getTable(String tableName) {
+        return null;
+    }
+
+    /**
+     * Queries and returns entities in the given table using the select and filter strings
+     *
+     * @param top          odata top parameter
```

Comment:
We don't align parameter names. If you edit your Settings, Code Style, Java, you can uncheck "Align parameter description" and "Align thrown exception description".

https://www.jetbrains.com/help/idea/code-style-java.html#javadoc